indexer-app/src/cmr/indexer/data/concepts/collection.clj

```diff
@@ -42,6 +43,10 @@
     [cmr.umm.collection.entry-id :as eid]
     [cmr.umm.umm-collection :as umm-c]))
 
+(defconfig cwic-tag
```

Comment:
I like Duncan's suggestion of putting any configuration that is used by two different apps into a new namespace `cmr.common-app.config` so it is not duplicated.